config/application.rb

```diff
@@ -29,6 +29,8 @@ class Application < Rails::Application
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
 
+    config.load_defaults 5.0
```

Comment:
hm, i worried about this. lemme take a look at what this would have been doing if we weren't calling it but we were using 5.2

Reply:
okay! based on this https://guides.rubyonrails.org/configuring.html#results-of-config-load-defaults and conversation with the folks that have done BMT rails upgrades, we feel good about this change to load_defaults 5.2